helm/ansiblelifecycledriver/templates/deployment.yaml

```diff
@@ -1,9 +1,12 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
```

Comment:
We need support for Kubernetes 1.13 onwards (to allow it to run in e.g. AIO as well as OCP) so some kind of switch could be used here - see the comment https://github.com/IBM/ansible-lifecycle-driver/issues/87#issue-662913902

Reply:
Seems like Kubernetes is smart enough to change this on the fly, depending on the version. Confirmed that it works ok in AIO (K8s 1.13) with these changes.